pylearner.ts

```diff
@@ -4724,8 +4952,739 @@ assert fibonacci(4) == 3
 assert fibonacci(5) == 5
 assert fibonacci(6) == 8`,
 expression: `fibonacci(7)`
-}
+}, {
```

Comment:
Verplaats de aliasing test cases naar `hiddenTests` onderaan; voorlopig wil ik de studenten van MI, die deze tool gebruiken, nog niet lastigvallen met list mutations, en zeker niet met alle test cases! Op termijn kunnen we, indien er nog tijd is, en nadat we de andere list mutations geïmplementeerd hebben, documentatie toevoegen over list mutations en onderaan enkele voorbeelden met list mutations aan `examples` toevoegen, maar dus niet in deze PR.

Reply:
Je hebt de test cases nog niet verplaatst?

Reply:
Was het vergeten. Nu opgelost. Heb wel gemerkt toen ik deze tests naar de hidden tests verplaatste dat ik de `# Wet Uitgesteld: b` die ik eerst in het begin formuleerde bij elk bewijs, waar ik het gebruikte, moest verplaatsen naar het einde van de `declaraties.` Bij de hidden tests gaf zo'n test case zonder wijzigingen plots een foutmelding.

Reply:
Dat komt omdat de `def`-regel niet geïndenteerd mag zijn. Verplaats de wetten weer naar bovenaan en zorg dat er geen spaties staan vóór het `def`-sleutelwoord.